packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -30,6 +36,8 @@ let container;
 let buffer = '';
 let hasErrored = false;
 let fatalError = undefined;
+let renderOptions;
+const rollupCache: Map<string, string | null> = new Map();
```

Comment:
This should just live in the test helper module.

Reply:
Import the compiled code string from the test helper module and keep all the deduping stuff isolated to that module.